### PR TITLE
Feature/parity etherium node

### PR DESCRIPTION
**Pull request by netoneko**
- [x] Add deployment script for Parity node
- [x] Add test script for Parity node
- [x] Inject env variable into sidechain-connector

### CODE REVIEW

File: deploy/cloudformation/README.md [hunk 1]
```diff
@@ -23,3 +23,17 @@ Deployment script creates specific parameters for each stack (common `parameters
...
 ./test.sh
 ```
+
+## Parity node
+
+Parity Etherium node is not part of ORBS network and is deployed as a separate service.
+
+TODO: extract to a different repo.
+
+```
```
Comment (lbeder) [inline deploy/cloudformation/README.md]: Could you annotate as ```bash?

File: deploy/cloudformation/create-stacks.parity.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -e
+
+aws cloudformation create-stack --template-body file://`pwd`/cloudformation.yaml --parameters "$(cat parameters.parity.json)" --stack-name orbs-network-parity
```
Comment (lbeder) [inline deploy/cloudformation/create-stacks.parity.sh]: Is there a way to parametrize on which network the Parity node runs? E.g., `parity` vs. `parity --chain=ropsten`?

File: deploy/cloudformation/parity_test.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const Web3 = require('web3');
```
Comment (lbeder) [inline deploy/cloudformation/parity_test.js]: Can you rename to `parity-test.js`?

File: deploy/cloudformation/parity_test.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+const web3 = new Web3(new Web3.providers.HttpProvider("http://ethereum.services.orbs-test.com:8545"));
```
Comment (lbeder) [inline deploy/cloudformation/parity_test.js]: Every federation member will have its own URL, so this should be parametrized as well.

File: deploy/cloudformation/test.parity.sh [hunk 5]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (lbeder) [inline deploy/cloudformation/test.parity.sh]: `#!/bin/bash -e`

File: deploy/parity/bootstrap.sh [hunk 6]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```
Comment (lbeder) [inline deploy/parity/bootstrap.sh]: #!/bin/bash -e

File: deploy/cloudformation/parity_test.js (hunk 4 above)
Comment (netoneko) [reply deploy/cloudformation/parity_test.js]: We decided it's going to be a part of different effort when we are preparing for alpha release.

File: deploy/bootstrap/.env [hunk 7]
```diff
@@ -1,3 +1,4 @@
 GOSSIP_PEERS=ws://bliny.nodes.orbs-test.com:60001,ws://pelmeni.nodes.orbs-test.com:60001,ws://borscht.nodes.orbs-test.com:60001,ws://pirogi.nodes.orbs-test.com:60001,ws://oladyi.nodes.orbs-test.com:60001,ws://olivier.nodes.orbs-test.com:60001,ws://solyanka.nodes.orbs-test.com:60001
 E2E_PUBLIC_API_ENDPOINT=0.0.0.0:51151
 E2E_NO_DEPLOY=true
+ethereumNodeAddress=http://ethereum.services.orbs-test.com:8545
```
[the remark this answers is not in the document]
Comment (netoneko) [reply deploy/bootstrap/.env]: I decided to move it to `staging.json`.

File: deploy/cloudformation/README.md (hunk 1 above)
Comment (netoneko) [reply deploy/cloudformation/README.md]: Done.

File: deploy/cloudformation/parity_test.js (hunk 3 above)
Comment (netoneko) [reply deploy/cloudformation/parity_test.js]: Done

File: deploy/cloudformation/test.parity.sh (hunk 5 above)
Comment (netoneko) [reply deploy/cloudformation/test.parity.sh]: Done.

File: deploy/parity/bootstrap.sh (hunk 6 above)
Comment (netoneko) [reply deploy/parity/bootstrap.sh]: Done.

File: deploy/cloudformation/create-stacks.parity.sh (hunk 2 above)
Comment (netoneko) [reply deploy/cloudformation/create-stacks.parity.sh]: I think we should wait a bit with configurability or load balanced setups.

**Comment by lbeder** [comment]
@netoneko can you add/document the missing issues in Leankit? Specifically: 1. Load balancing+multiple nodes; 2. Testnet/mainnest configurability?

**Comment by netoneko** [comment]
@lbeder done.

**Pull request merged**